### DISCUSSION TITLE
смена категорий в getCategoryTracks

### DISCUSSION THREAD

**Discussion by amd64fox** [Chimildic/goofy#186] [Помощь с ошибкой] [answered]
Я так понял некоторые категории стали прилетать не как имя, а как id, например сейчас у меня категория `metal` прилетает в таком виде `0JQ5DAqbMKFDkd668ypn6O`.

Не знаю навсегда так стало или временно, но сам факт имеет место.

**Comment by Chimildic** [comment, marked as answer]
Чем-то мешает? Они наверное константы, хватит один раз подставить и комментарий для напоминалки дописать.

**Reply by amd64fox** [reply]
У себя логику кода изменил под это. Подумал если новые люди будут смотреть полный список категорий через вариант из справки
```javascript
let listCategory = Source.getListCategory({ limit: 50, country: 'US' });
console.log(listCategory.map(c => c.id).join('\n'));
```
то для них не совсем понятно как определить по этим константам какая именно это категория, можно чуть подправить и добавить рядом имя, например так 
```javascript
let listCategory = Source.getListCategory({ limit: 50, country: 'US' });
console.log(listCategory.map(c => '\n' + c.name + '\n' + c.id).join('\n'));
 ```
